core/controllers/story_editor.py

```diff
@@ -154,4 +154,3 @@ def delete(self, topic_id, story_id):
                 Exception('The topic with the given id doesn\'t exist.'))
 
         story_services.delete_story(self.user_id, story_id)
```

Comment:
Shouldn't deleting a story also include removing it from the relevant topic? /cc @tjiang11